client/components/account-status/account-fees/index.js

```diff
@@ -51,11 +51,15 @@ const AccountFees = ( props ) => {
 			{ feeDescription }
 			<ExpirationBar
 				feeData={ currentFee }
-				currencyCode={ currencyCode }
+				currencyCode={
+					currentFee.volume_currency ?? currentFee.currency
+				}
 			/>
 			<ExpirationDescription
 				feeData={ currentFee }
-				currencyCode={ currencyCode }
+				currencyCode={
+					currentFee.volume_currency ?? currentFee.currency
+				}
```

Comment:
Same here: can we move the logic into `ExpirationDescription`?

Reply:
Addressed in 1921d85